openslides/motions/static/js/motions/site.js

```diff
@@ -21,7 +21,7 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
                 * other project that might want to use this HTML to PDF parser.
                 * https://github.com/OpenSlides/OpenSlides/issues/2361
                 */
-                return converter.convertHTML($scope.lineBrokenText, $scope);
+                return converter.convertHTML(motion.getTextWithLineBreaks($scope.version, 0), $scope);
```

Comment:
from
https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/base.js#L184

```
getTextWithLineBreaks: function (versionId, highlight, callback) {
    var lineLength = Config.get('motions_line_length').value,
    html = this.getVersion(versionId).text;

    return lineNumberingService.insertLineNumbers(html, lineLength, highlight, callback);
},
```
We use the given `$scope.version` and no (`0`) highlighting and no callback function